### ISSUE TITLE
GCC 8 fails to build 2 e3sm_developer tests on anlworkstation

### ISSUE THREAD

**Issue by dqwu** [E3SM-Project/E3SM#3169]
This build issue is caused by PR #2989 on seaice update.

SMS.T62_oQU120_ais20.MPAS_LISIO_TEST has build errors like below:
```
mpas_seaice_advection_incremental_remap_tracers.f90:1035:34:

           thisTracer % array2D => tracerField2DReal % array
                                  1
Error: Assignment to contiguous pointer from non-contiguous target at (1)
```
https://github.com/MPAS-Dev/MPAS-Model/issues/342 has already reported similar build errors (Intel v19 in debug mode on PNNL Compy cluster), which will be fixed by https://github.com/MPAS-Dev/MPAS-Model/pull/343

SMS_P12x2.ne4_oQU240.A_WCYCL1850.anlworkstation_gnu.allactive-mach_mods has build errors like below:
```
mpas_seaice_velocity_solver_constitutive_relation.f90:174:51:

 !$omp declare simd(seaice_evp_constitutive_relation)
                                                   1
Error: Symbol ‘seaice_evp_constitutive_relation’ at (1) has already been host associated
```
@amametjanov 
You can run following commands to reproduce the build errors on anlworkstation.
```
git clone --branch dqwu/anlworkstation_gcc8 https://github.com/E3SM-Project/E3SM.git
cd E3SM
git submodule update --init
cd cime/scripts
./create_test SMS.T62_oQU120_ais20.MPAS_LISIO_TEST --no-run --machine=anlworkstation
./create_test SMS_P12x2.ne4_oQU240.A_WCYCL1850.anlworkstation_gnu.allactive-mach_mods --no-run --machine=anlworkstation
```

**Comment by amametjanov** [comment]
Yes, gnu apparently does not support `omp declare simd` and the build issue can be avoided by MPAS-Dev/MPAS-Model#348.

**Comment by dqwu** [comment]
@amametjanov 
With latest next branch (mpas-source submodule has been updated), these 2 tests passed.
Therefore, this issue is ready to be closed, thanks.

**Comment by amametjanov** [comment]
Fixed by E3SM-Project/E3SM#3230
